receiver/statsdreceiver/receiver.go

```diff
@@ -90,7 +92,7 @@ func (r *statsdReceiver) Start(ctx context.Context, host component.Host) error {
 	ticker := time.NewTicker(r.config.AggregationInterval)
 	r.parser.Initialize(r.config.EnableMetricType, r.config.IsMonotonicCounter, r.config.TimerHistogramMapping)
 	go func() {
-		if err := r.server.ListenAndServe(r.parser, r.nextConsumer, r.reporter, transferChan); err != nil {
+		if err := r.server.ListenAndServe(r.parser, r.nextConsumer, r.reporter, transferChan); !errors.Is(err, net.ErrClosed) {
```

Comment:
Also http.ErrServerClosed is the same correct?

Reply:
Also != nil still needed

Reply:
> Also != nil still needed

Done.

> Also http.ErrServerClosed is the same correct?

In the statsd context, I believe we have a custom implementation of the UDP server. So it is different from `http.ErrServerClosed`.